reference/forms/types/options/max_length.rst.inc

```diff
@@ -1,11 +1,11 @@
+max_length
+~~~~~~~~~~
+
 .. caution::
 
     The ``max_length`` option has been deprecated and will be removed in 3.0.
```

Comment:
Good point, I added the missing information and unified the wording a bit where necessary.